en/docs/learn/api-security/oauth2/grant-types/authorization-code-grant.md

```diff
@@ -63,7 +63,7 @@ The steps below show how access tokens are generated for the authorization code
 !!! note
     **Before you begin**
 
-    The following instructions use the sample playground webapp. For instructions on how to set up the sample webapp, see [Setting up the Sample Webapp](https://is.docs.wso2.com/en/5.9.0/develop/deploying-the-sample-app/) .
+    The following instructions use the sample playground webapp. For instructions on how to set up the sample webapp, see [Setting up the Sample Webapp](https://is.docs.wso2.com/en/latest/learn/deploying-the-sample-app/#deploying-the-playground2-webapp) .
```

Comment:
@tgtshanika , shouldn't this URL be https://is.docs.wso2.com/en/5.10.0/learn/deploying-the-sample-app/#deploying-the-playground2-webapp instead of pointing to latest/..... as API-M 3.1.0 is compatible with 5.10.0. Whereas latest/... will change as the new IS product versions come out.

Reply:
Fixed